src/main/java/org/graylog/integrations/aws/resources/KinesisSetupResource.java

```diff
@@ -70,10 +70,9 @@ public KinesisNewStreamResponse createNewKinesisStream(@ApiParam(name = "JSON bo
         final User user = getCurrentUser();
         LOG.info("User [{}] agreed to the Kinesis auto-setup, which will create a Kinesis stream [{}], " +
                  "role/policy, and a CloudWatch log group Subscription. " +
-                 "This has been recorded, since AWS resources that will cost money are being created " +
-                 "at the sole request of this user.", user.getId(), request.streamName());
+                 "This has been recorded as the listed user has accepted the responsibility in associated potentially " +
```

Comment:
This looks great to me. Just wondering if we can add a comma after the word "recorded"
```suggestion
                 "This has been recorded, as the listed user has accepted the responsibility in associated potentially " +
```